src/convertToObject.js

```diff
@@ -6,7 +6,25 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  const stylesObject = {};
+
+  const declarations = sourceString.split(';');
+
+  declarations.forEach((declaration) => {
+    const trimmedDeclaration = declaration.trim();
+
+    if (!trimmedDeclaration) {
+      return;
+    }
+
+    const [property, value] = trimmedDeclaration
+      .split(':')
+      .map((part) => part.trim());
```

Comment:
[CODE STYLE] - All chained methods called on the same level should have the same indentation to improve readability. Consider formatting the chained `split` and `map` calls with proper indentation.